src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -115,6 +134,10 @@ public static void initConfig(String configFile) throws JSONException, IOExcepti
     } else if (tmpCommunicationType.equalsIgnoreCase(post2019)
         || tmpCommunicationType.equalsIgnoreCase(secondPost2019)) {
       communicationType = omf;
+    } else if (tmpCommunicationType.equalsIgnoreCase(omfOcs)) {
+      communicationType = OCS;
+      setOCSToken();
+      Logger.LOG_DEBUG("using OMF OCS");

```

Comment:
This comment could probably use some more context. Also should there be a similar comment for post2019 and pre2019?

Reply:
added in a new commit